app/src/main/java/org/jellyfin/androidtv/details/PersonPresenter.kt

```diff
@@ -1,28 +1,42 @@
 package org.jellyfin.androidtv.details
 
+import android.content.Context
+import android.content.Intent
 import android.graphics.drawable.BitmapDrawable
 import android.view.ViewGroup
+import androidx.appcompat.view.ContextThemeWrapper
 import androidx.leanback.widget.ImageCardView
-import androidx.leanback.widget.Presenter
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.GlobalScope
 import kotlinx.coroutines.launch
 import org.jellyfin.androidtv.R
 import org.jellyfin.androidtv.TvApp
-import org.jellyfin.androidtv.model.itemtypes.Person
-import org.jellyfin.androidtv.presentation.MyImageCardView
+import org.jellyfin.androidtv.base.ClickablePresenter
+import org.jellyfin.androidtv.model.itemtypes.BriefPersonData
 
 private const val LOG_TAG = "PersonPresenter"
 
-class PersonPresenter : Presenter() {
+class PersonPresenter(private val context: Context) : ClickablePresenter() {
+
+	override fun onItemClicked(item: Any?) {
+		requireNotNull(item)
+		val person = item as BriefPersonData
+		val intent = Intent(context, FullDetailsActivity::class.java)
+		intent.putExtra("ItemId", person.id)
+
+		context.startActivity(intent)
+	}
+
 	override fun onCreateViewHolder(parent: ViewGroup?): ViewHolder {
-		return ViewHolder(ImageCardView(parent!!.context))
+		return ViewHolder(ImageCardView(ContextThemeWrapper(parent!!.context, R.style.MarqueeImageCardViewTheme)))
 	}
 
 	override fun onBindViewHolder(viewHolder: ViewHolder?, item: Any?) {
```

Comment:
`override fun onBindViewHolder(viewHolder: ViewHolder, item: Any) {`
should work, so you won't need the `requireNotNull` calls